index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header>
+   <nav class="top-header">
+     <a class="welcome identical-to" href="#">Welcome</a>
+      <span class="identical-to">&#8801;</span>
+  </nav>
+  </header> 
+  <main>
+    <section>
+    <div class="intro">
+    <h1 class="first-name">Hey there. I'm Stephen</h1>
+    <h1 class="first-name software-dev">I'm a software developer</h1>
```

Comment:
- [ ] You should replace this `<h1>` tag with `<h2>, <h3> .. <h6>` for the `I'm a software developer` text as it was indicated by the previous reviewer for semantic reasons. As a general rule, you need to have only one `<h1>` tag in an html document assigned to the top most heading in your page.

Reply:
Thank you. This was not what the previous review recommended. The tex was originally in h2, he asked to change it to h1. Now you are asking me to change back to h2. Please, this change is not necessary. Read thee complete former review that you cited 

"- [x] The Html file structure stipulates that we use an `h1` tag for the first header of our page, then we can use `h2`, `h3` tags, and so on for the following headers after the `h1` tag.
This practice helps make our HTML files more readable to browsers, kindly change the `h2` tags to `h1` tags.

_Originally posted by @Munsa1 in https://github.com/stephnna/portfolio/pull/2#discussion_r856478642_"